applications/nrf_desktop/src/modules/Kconfig.dvfs

```diff
@@ -7,13 +7,28 @@
 menuconfig DESKTOP_DVFS
 	bool "DVFS module"
 	depends on SOC_NRF54H20_CPUAPP
+	select NRFS_DVFS_SERVICE_ENABLED
+	select CLOCK_CONTROL
+	select CLOCK_CONTROL_NRF2
 	default y
 	help
 	  This option enable DVFS module which switches frequency and voltage according
 	  to application needs.
 
 if DESKTOP_DVFS
 
+config DESKTOP_DVFS_FREQ_HIGH
+	int
+	default 320000000
+
+config DESKTOP_DVFS_FREQ_MED
+	int
+	default 128000000
+
+config DESKTOP_DVFS_FREQ_LOW
+	int
+	default 64000000
```

Comment:
@gmarull these should come from dts?

Reply:
But we don't have those frequencies in dts

Reply:
The range of valid frequencies should definitely be in DT, this is a HW capability thing. The software choices are fine in Kconfig. However, adjusting frequencies directly in applications doesn't seem like a great design. This should be _governed_ by some framework.

Reply:
Ok, but valid frequencies should be added globally in dts at board level. Till it is not added shouldn't we base on frequencies from Kconfig?

>> However, adjusting frequencies directly in applications doesn't seem like a great design. 

Object of this task is to move responsibility of changing frequencies by clock_api instead directly from application (module has been written prior to clock driver)

Reply:
@nordicjm @gmarull  lets merge as it is for 2.8 release and change to dts later when those frequencies will be properly added to dts.